cluster-autoscaler/cloudprovider/gce/gce_manager.go

```diff
@@ -220,7 +218,7 @@ func CreateGceManager(configReader io.Reader, mode GcpCloudProviderMode, cluster
 		gceService:  gceService,
 		migCache:    make(map[GceRef]*Mig),
 		location:    location,
-		isRegional:  isRegional,
+		isRegional:  regional,
```

Comment:
nit: let's make this consistent and go for one of isRegional and regional in both places.

Reply:
Done. Went with `regional` following the AllElseBeingEqualLessCamelCaseIsBetter principle;)